proto/cosmos/accounts/vesting/v1/tx.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+package cosmos.accounts.vesting.v1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos_proto/cosmos.proto";
+import "cosmos/accounts/vesting/v1/vesting.proto";
+import "amino/amino.proto";
+
+option go_package = "cosmossdk.io/x/vesting/types/v1";
+
+//-------------------------------------- INIT --------------------------------------
+
+// MsgInitVestingAccount defines a message that enables creating a vesting
+// account.
+message MsgInitVestingAccount {
+  option (amino.name) = "cosmos-sdk/MsgInitVestingAccount";
+
+  option (gogoproto.equal) = true;
+
+  // owner of the vesting account
+  string   owner                           = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  repeated cosmos.base.v1beta1.Coin amount = 2 [
+    (gogoproto.nullable)     = false,
+    (amino.dont_omitempty)   = true,
+    (amino.encoding)         = "legacy_coins",
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+  ];
+
+  // end of vesting as unix time (in seconds).
+  int64 end_time = 3;
+  // start of vesting as unix time (in seconds).
+  int64 start_time = 4;
+}
+
+// MsgInitVestingAccountResponse defines the Msg/InitVestingAccount response type.
+message MsgInitVestingAccountResponse {}
```

Comment:
should we return the address that it was created under?

Reply:
hmmmm make sense i will add it in